easybuild/easyconfigs/l/LAMMPS/LAMMPS-patch_4Feb2020-foss-2019a-Python-3.7.2-CUDA-10.1.105-kokkos.eb

```diff
@@ -0,0 +1,121 @@
+name = 'LAMMPS'
+version = 'patch_4Feb2020'
+local_cuda_ver = '10.1.105'
+local_python_versionsuffix = '-Python-%(pyver)s'
+versionsuffix = local_python_versionsuffix
+versionsuffix += '-CUDA-%s' % local_cuda_ver
+versionsuffix += '-kokkos'
+
+homepage = 'https://lammps.sandia.gov/'
+description = """LAMMPS is a classical molecular dynamics code, and an acronym
+for Large-scale Atomic/Molecular Massively Parallel Simulator. LAMMPS has
+potentials for solid-state materials (metals, semiconductors) and soft matter
+(biomolecules, polymers) and coarse-grained or mesoscopic systems. It can be
+used to model atoms or, more generically, as a parallel particle simulator at
+the atomic, meso, or continuum scale. LAMMPS runs on single processors or in
+parallel using message-passing techniques and a spatial-decomposition of the
+simulation domain. The code is designed to be easy to modify or extend with new
+functionality.
+"""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'openmp': True, 'usempi': True, 'cstd': 'c++11'}
+
+# 'https://github.com/lammps/lammps/archive/'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['%(version)s.tar.gz']
+checksums = ['3bf3de546ede34ffcd89f1fca5fd66aa78c662e7c8a76e30ce593e44a00d23ce']
+
+local_source_dir_name = '%(namelower)s-%(version)s'
+
+builddependencies = [
+    ('CMake', '3.13.3'),
+    ('pkg-config', '0.29.2'),
+    ('archspec', '0.1.0', '', True),
+]
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('libpng', '1.6.36'),
+    ('libjpeg-turbo', '2.0.2'),
+    ('netCDF', '4.6.2'),
+    ('GSL', '2.5'),
+    ('zlib', '1.2.11'),
+    ('gzip', '1.10'),
+    ('HDF5', '1.10.5'),
+    ('tbb', '2019_U4'),
+    ('PCRE', '8.43'),
+    ('libxml2', '2.9.8'),
+    ('FFmpeg', '4.1.3'),
+    ('Voro++', '0.4.6'),
+    ('FFTW', '3.3.8'),
+    ('kim-api', '2.1.2'),
+    ('Eigen', '3.3.7', '', True),
+    ('yaff', '1.6.0', local_python_versionsuffix),
+    ('PLUMED', '2.5.3', local_python_versionsuffix),
+    ('CUDA', local_cuda_ver, '', True),
+]
+
+# Lammps requires separate build dir, ends up in error if not.
+separate_build_dir = True
+
+# For configuration options for cmake, see:
+# https://github.com/lammps/lammps/blob/master/cmake/README.md#reference
+
+preconfigopts = 'unset LIBS && '
+prebuildopts = preconfigopts
+
+# LAMMPS Configuration Options
+# https://github.com/lammps/lammps/blob/master/cmake/README.md#lammps-configuration-options
+configopts = " "
```

Comment:
This can be removed?

Maybe just add a comment like "To use additional custom configuration options, use the 'configopts' easyconfig parameter"